src/course-unit/CourseUnit.jsx

```diff
@@ -142,28 +146,28 @@ const CourseUnit = ({ courseId }) => {
               />
             )}
             breadcrumbs={(
-              <Breadcrumbs />
+              <Breadcrumbs
+                courseId={courseId}
+                sequenceId={sequenceId}
+              />
             )}
             headerActions={(
               <HeaderNavigations
+                unitCategory={unitCategory}
                 headerNavigationsActions={headerNavigationsActions}
               />
             )}
           />
-          <Sequence
-            courseId={courseId}
-            sequenceId={sequenceId}
-            unitId={blockId}
-            handleCreateNewCourseXBlock={handleCreateNewCourseXBlock}
-            showPasteUnit={showPasteUnit}
-          />
-          <Layout
-            lg={[{ span: 8 }, { span: 4 }]}
-            md={[{ span: 8 }, { span: 4 }]}
-            sm={[{ span: 8 }, { span: 3 }]}
-            xs={[{ span: 9 }, { span: 3 }]}
-            xl={[{ span: 9 }, { span: 3 }]}
-          >
+          {isUnitVerticalType && (
+            <Sequence
+              courseId={courseId}
+              sequenceId={sequenceId}
+              unitId={blockId}
+              handleCreateNewCourseXBlock={handleCreateNewCourseXBlock}
+              showPasteUnit={showPasteUnit}
+            />
+          )}
+          <Layout {...layoutGrid}>
```

Comment:
There are some console errors that only appear on the library container page, but not on the normal unit page:

```
CourseUnit.jsx:170 Warning: Failed prop type: Invalid prop `md` of type `object` supplied to `ForwardRef(_c3)`, expected an array.
    at _c3 (webpack-internal:///./node_modules/@openedx/paragon/dist/Layout/index.js:125:23)
    at CourseUnit (webpack-internal:///./src/course-unit/CourseUnit.jsx:93:5)

thunk.js:81 xl prop accepts array which length must be equal to the number of children. Error Component Stack
    at _c3 (index.js:125:23)
    at section (<anonymous>)
    at div (<anonymous>)
    at eval (Container.js:21:23)
    at _c (index.js:125:19)
    at CourseUnit (CourseUnit.jsx:41:23)
```

Reply:
Refactored hook `useLayoutGrid` and fix this issue